libs/error-analysis/src/lib/ErrorAnalysisDashboard/Controls/TreeViewRenderer/TreeViewUtils.ts

```diff
@@ -0,0 +1,41 @@
+import { ColorPalette } from "./../../ColorPalette";
+
+export function nodeColor(isErrorMetric: boolean, perc: number): string {
+  if (perc == 0) {
+    return ColorPalette.white;
+  }
+  if (isErrorMetric) {
+    if (perc > 0.75) {

```

Comment:
nit: will changing it to a `switch` statement be better?

Reply:
thanks tong, the file is not needed anymore.